### ISSUE TITLE
City of Turku open data layers

### ISSUE THREAD

**Issue by ilmaisin** [osmlab/editor-layer-index#751]
I would like to have the layers named "Turun asemakaava" (Zoning Plan of Turku) and "Turun maastokartta" (Terrain Map of Turku) from the WMS Open Data feed of City of Turku.

## URL of the imagery
https://opaskartta.turku.fi/TeklaOGCWeb/WMS.ashx
Description page: https://www.turku.fi/turku-tieto/kartat-ja-paikkatieto/karttapalveluiden-rajapinnat

## Two letter country code
fi

## Desired layer name
Turun asemakaava
Turun maastokartta

## License
(required)<br/>
CC-BY 4.0 with OSM waiver (http://www.turku.fi/avoindata/jhs189)

## Is the license compatible with OSM?
Yes

## Attribution (text, URL, required?)
Mention in "[Contributors](https://wiki.openstreetmap.org/wiki/Contributors)" wiki page enough

## Privacy policy
https://www.turku.fi/turku-tieto/tietosuoja/kayttotiedot-ja-evasteet

**Comment by grischard** [comment]
Thank you! Do you want to have a go at doing a geojson? I'll help you out if you get stuck.

Also, we'll need a geometry and a link to a privacy policy.

**Comment by ilmaisin** [comment]
I added the privacy policy to the issue. I'll need to investigate that geometry thing little further since the larger scale layers have much less coverage than the smaller ones.

**Comment by theel0ja** [comment]
> CC-BY 4.0 with OSM waiver (http://www.turku.fi/avoindata/jhs189)

Does this change anything with #537?

**Comment by ilmaisin** [comment]
> > CC-BY 4.0 with OSM waiver (http://www.turku.fi/avoindata/jhs189)
> 
> Does this change anything with #537?

Yes, the waiver has been granted since the filing of that issue. I commented it there too.

**Comment by ilmaisin** [comment]
Here are polygons for asemakaava 
([turun-asemakaava-polygon.geojson.txt](https://github.com/osmlab/editor-layer-index/files/4122051/turun-asemakaava-polygon.geojson.txt)) and maastokartta ([turun-maastokartta-polygon.geojson.txt](https://github.com/osmlab/editor-layer-index/files/4122053/turun-maastokartta-polygon.geojson.txt)) respectively.

**Comment by ilmaisin** [comment]
Or when I look at the other polygons in the repository, it seems that these simplified polygons could be more appropriate:
[turun-asemakaava-polygon-simple.geojson.txt](https://github.com/osmlab/editor-layer-index/files/4122160/turun-asemakaava-polygon-simple.geojson.txt)
[turun-maastokartta-polygon-simple.geojson.txt](https://github.com/osmlab/editor-layer-index/files/4122162/turun-maastokartta-polygon-simple.geojson.txt)

**Comment by grischard** [comment]
Yes, simplified polygons are better - it's just to know where to show the layers as an available option, and every OSM editor around the globe has to download that polygon.

**Comment by larmax** [comment]
Hi, could the layer called "Turun Ilmakuva 2021" (Turku aerial imagery 2021) be added? It shouldn't replace the old one as that covers a larger area. Not sure how to get the bbox though.

**Comment by npsalo** [comment]
Hi, could the layer called "Turunseutu ilmakuva 2023" (Turku area aerial imagery 2023) be added? Not sure how to get the bbox though.
